### ISSUE TITLE
[eBUG] Error saving player in the db with an apostrophe in the surname

### ISSUE THREAD

**Issue by tychobrailleur** [akasolace/HO#490]
**Describe the bug**
A player whose surname has an apostrophe cannot be saved in the db (e.g. O'Kennedy):

```
 [Error]   JDBCAdapter: JDBCAdapter.executeUpdate : java.sql.SQLSyntaxErrorException: unexpected token: KENNEDY required: )
Statement: INSERT INTO SPIELER ( GelbeKarten , SpielerID , FirstName , NickName, LastName , Age , AgeDays , Kondition , Form , Torwart , Verteidigung , Spielaufbau , Fluegel , Torschuss , Passpiel , Standards , SubTorwart , SubVerteidigung , SubSpielaufbau , SubFluegel , SubTorschuss , SubPasspiel , SubStandards , OffsetTorwart , OffsetVerteidigung , OffsetSpielaufbau , OffsetFluegel , OffsetTorschuss , OffsetPasspiel , OffsetStandards , iSpezialitaet , iCharakter , iAnsehen , iAgressivitaet , Fuehrung , Erfahrung , Gehalt , Bonus , Land , Marktwert , Verletzt , ToreFreund , ToreLiga , TorePokal , ToreGesamt , Hattrick , Bewertung , TrainerTyp, Trainer, HRF_ID, Datum, PlayerNumber, TransferListed,  Caps, CapsU20, TrainingBlock, Loyalty, HomeGrown, SubExperience, NationalTeamID ) VALUES(0,446958673,'Andrew','','O'Kennedy',20,102,7,7,1,5,9,2,3,6,6,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0,0,0,0,0,0,0,3,0,4,4,3,3,4500,0,16,5410,-1,1,1,0,4,0,0,-1,0,72,'2020-05-03 14:01:03.0',9,0,0,0,false,20,true,0.0,0)
java.sql.SQLSyntaxErrorException: unexpected token: KENNEDY required: )
	at org.hsqldb.jdbc.JDBCUtil.sqlException(Unknown Source)
	at org.hsqldb.jdbc.JDBCUtil.sqlException(Unknown Source)
	at org.hsqldb.jdbc.JDBCStatement.fetchResult(Unknown Source)
```

**To Reproduce**
0. Your team must have a player whose surname has an apostrophe
1. Download the details of players

**Screenshots**

**Platform information:**
 - OS: macOS
 - Version HEAD master

**Additional context**

**Comment by akasolace** [comment]
@tychobrailleur this one is annoying and needs to be fixed before next beta. Do you take care of it? If you want I can do it if you ok to share your db. Let me know ...

**Comment by tychobrailleur** [comment]
I can take care of it, hopefully later today.

**Comment by akasolace** [comment]
ok perfect

**Comment by akasolace** [comment]
closed by #493
